OKTA-494746 Update Okta Expression Language Overview for multiple AD instances (#3350)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

* Be sure to pass the correct App name for the `managerSource`, `assistantSource`, and `attributeSource` parameters.
* At this time, `active_directory` is the only supported value for `managerSource` and `assistantSource`.
* Calling the `getManagerUser("active_directory")` function doesn't trigger a user profile update after the manager is changed.

> * The manager and assistant functions are not supported for user profiles sourced from multiple Active Directory instances.

Reviewer: ```suggestion
* The manager and assistant functions aren't supported for user profiles sourced from multiple Active Directory instances.
```
